lib/elixir_internal_certification_web/controllers/api/v1/keyword_controller.ex

```diff
@@ -0,0 +1,54 @@
+defmodule ElixirInternalCertificationWeb.Api.V1.KeywordController do
+  use ElixirInternalCertificationWeb, :controller
+
+  alias ElixirInternalCertificationWeb.Api.ErrorView
+  alias ElixirInternalCertificationWeb.Helpers.KeywordHelper
+  alias Plug.Upload
+
+  @max_keywords_per_upload Application.compile_env!(
+                             :elixir_internal_certification,
+                             :max_keywords_per_upload
+                           )
+
+  def create(
+        %{assigns: %{current_user: current_user}} = conn,
+        %{"file" => %Upload{path: path} = _uploaded_file} = _params
```

Comment:
```suggestion
        %{"file" => %Upload{path: path}} = _params
```
Quite a lot of bloat otherwise?